test/unit/spec/nbextensions/appCell2/widgets/appCellWidget-spec.js

```diff
@@ -119,26 +118,39 @@ define([
         });
 
         it('Has expected functions when instantiated', () => {
-            expect(mockAppCell.init).toBeDefined();
-            expect(mockAppCell.attach).toBeDefined();
-            expect(mockAppCell.start).toBeDefined();
-            expect(mockAppCell.stop).toBeDefined();
-            expect(mockAppCell.detach).toBeDefined();
+            ['init', 'attach', 'start', 'stop', 'detach'].forEach(fn => {
+                expect(mockAppCell[fn]).toBeDefined();
+            });
         });
 
-        it('has a method "init" which returns a promise then null', async () => {
-            var initPromise = mockAppCell.init();
-            expect(initPromise instanceof Promise).toBeTrue();
+        it('has a method "init" which returns a promise then null', () => {
+            return mockAppCell.init()
+                .then(() => {
+                    // something to see if it worked
+                });
         });
 
         it('has a method stop which returns a Promise', () => {
-            var stopPromise = mockAppCell.stop();
-            expect(stopPromise instanceof Promise).toBeTrue();
+            return mockAppCell.init()
```

Comment:
Would you be able to change the variable name to `appCellWidgetInstance`, since they're not actually mocks?